source-jdbc/src/main/java/org/apache/kylin/source/jdbc/extensible/JdbcHiveInputBase.java

```diff
@@ -57,6 +58,9 @@ protected AbstractExecutable createSqoopToFlatHiveStep(String jobWorkingDir, Str
             KylinConfig config = flatDesc.getDataModel().getConfig();
             PartitionDesc partitionDesc = flatDesc.getDataModel().getPartitionDesc();
             String partCol = null;
+            boolean enableQuote = dataSource.getSqlConverter().getConfigurer().enableQuote();
+            SqlDialect sqlDialect = enableQuote ? dataSource.getSqlConverter().getConfigurer().getSqlDialect(): FlatTableSqlQuoteUtils.NON_QUOTE_DIALECT;
+            SqlConverter.IConfigurer iconfigurer = dataSource.getSqlConverter().getConfigurer();
```

Comment:
`IConfigurer config = dataSource.getSqlConverter().getConfigurer()` and reuse config to avoid unnecessary temporary variables and make code clean.